packages/convenience/contracts/Convenience.sol

```diff
@@ -230,6 +278,13 @@ contract Convenience is Ownable  {
     /// @param votingAppType Enumerated voting app type (primary or secondary)
     /// @param userAddress User address
     /// @param voteIds Array of vote IDs for which data will be retrieved
+    /// @return executed If the vote has been executed
+    /// @return yea Total voting power voted for "For"
```

Comment:
"In favor of" maybe better? "pro" is another alternative

Reply:
I wanted to use the exact same word from the dashboard. The voting modal options are For and Against there.